flex/http.py

```diff
@@ -15,6 +16,12 @@
 except ImportError:
     _tornado_available = False
 
+_falcon_available = True
```

Comment:
Sure, this was my original intention but I wanted to be consistent with `_tornado_available`. I can change both in a separate commit. What do you think?